ttyplot: add page (#6679)
tldr-pages/tldr: pages/linux/ttyplot.md

# ttyplot

> > A realtime plotting utility for terminal with data input from stdin / pipe.

Reviewer: pipe and stdin is basically the same in this case
```suggestion
> A realtime plotting utility for the command line with data input from stdin.
```

> More information: <https://github.com/tenox7/ttyplot>.

- Plot the values `1`, `2` and `3` (`cat` prevents ttyplot to exit):

> `{ echo 1 2 3; cat } | ttyplot`

Reviewer: ```suggestion
`{ echo {{1 2 3}}; cat } | ttyplot`
```

- Set a specific title and unit:

> `{ echo 1 2 3; cat } | ttyplot -t {{title}} -u {{unit}}`

Reviewer: ```suggestion
`{ echo {{1 2 3}}; cat } | ttyplot -t {{title}} -u {{unit}}`
```

- Use a while loop to continuously plot random values:

> `{ while true; do echo $RANDOM; sleep 1; done } | ttyplot`

Reviewer: ```suggestion
`{ while {{true}}; do echo {{$RANDOM}}; sleep {{1}}; done } | ttyplot`
```

> - Parse the output from other commands and visualize it:

Reviewer: This specific regex probably only works for ping, so we shouldn't make the description too general
```suggestion
- Parse the output from `ping` and visualize it:
```

> `ping 8.8.8.8 | sed -u 's/^.*time=//g; s/ ms//g' | ttyplot -t "ping to 8.8.8.8" -u ms`

Reviewer: ```suggestion
`ping {{8.8.8.8}} | sed -u '{{s/^.*time=//g; s/ ms//g}}' | ttyplot -t "{{ping to 8.8.8.8}}" -u {{ms}}`
```
